README_GPDB_ru.md

```diff
@@ -0,0 +1,207 @@
+# 1. Поддерживаемые версии Greenplum
+
+На данный момент поддерживается только Greenplum 6.
```

Comment:
It might be good to place a reference to gpdb 6 documentation here. It is 'nice to have', if you do not see any drawbacks.